Entity/Profs.py

```diff
@@ -7,11 +7,11 @@
 
 class Profs(Base):
     __tablename__ = "Profs"
```

Comment:
We need an id column to keep the pattern we have in other entities so insert/update_entity will still work for this Entity